### ISSUE TITLE
Setting Zigbee color temperature breaks with 3.0.0

### ISSUE THREAD

**Issue by agross** [ioBroker/ioBroker.lovelace#397]
**Describe the bug**  
If version 3.0.0 is installed, I can no longer set the color temperature for my Zigbee lights. Setting the color still works.

**To Reproduce**  
1. Install 3.0.0
2. Configure a Zigbee light capable of setting the color temperature
3. Drag the color temperature slider
4. The color temperature state does not change

**Expected behavior**  
The color temperature state should be updated.

**Screenshots & Logfiles**  

These are the attributes of one such light (same as before as far as I can tell).

```
brightness: zigbee.0.00178801082e99a8.brightness
color_mode: rgb
color_temp: zigbee.0.00178801082e99a8.colortemp
entity_picture: ./adapter/zigbee/img/philips_hue_color.png
friendly_name: Living Room #1
max_mireds: 450
min_mireds: 153
rgb_color: zigbee.0.00178801082e99a8.color
supported_color_modes: onoff,color_temp,brightness,rgb
```

**Versions:**  
 - Adapter version: 3.0.0
 - JS-Controller version: latest
 - Node version: 16.16
 - Operating system: Raspbian/docker

**Additional context**  

With 3.0.0 installed I got the chance to test the color temp slider while the Browser still had 2.x loaded (notification "There is a new version of the frontend available" is shown). Dragging the slider worked. I then reloaded the Browser, the slider no longer had any effect.

**Comment by agross** [comment]
Hm, I just installed from GitHub. Is it expected that the entity attributes change and now should include `color_temp_kelvin`. I'm asking because there still only `color_temp` and the color temp slider does not work.

**Comment by Garfonso** [comment]
Hm... why is this auto closed now? Just because of a commit comment? Strange. :-/

On topic:
In the past the frontend only supported mired as color temperature. This is still there in the attributes, but the frontend seems to have completely switched to kelvin now (at least the build in stuff... hm.. maybe some custom card's did not and I should just fill/support both?) and this is now in `color_temp_kelvin`, yes.

**Comment by Garfonso** [comment]
I re-added color_temp / mired support. So now both are supported. Both are set from ioBroker and setting any of them in the frontend will control the ioBroker state associated. Maybe that helps?

**Comment by agross** [comment]
> Hm... why is this auto closed now? Just because of a commit comment? Strange. :-/

If you use `fixes #123` or `closes #123` (or [any magic word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)) in a commit message GitHub will close `#123` once the commit is reachable via the default branch (e.g. `master` by pushing or merging).

**Comment by agross** [comment]
Thanks, appears to be working now!
